src/Phan/Analysis/PostOrderAnalysisVisitor.php

```diff
@@ -5158,4 +5159,31 @@ public function visitThrow(Node $node): Context
 
         return $this->context;
     }
+
+    private function checkForAbstractPrivateMethodInTrait(Clazz $class, Method $method): void
+    {
+        // Skip PHP 8.0+
+        if (Config::get_closest_minimum_target_php_version_id() >= 80000) {
+            return;
+        }
+
+        if (! $class->isTrait()) {
+            return;
+        }
+
+        if (! $method->isPrivate()) {
+            return;
+        }
+
+        if (! $method->isAbstract()) {
```

Comment:
```suggestion
        if (!$method->isAbstract()) {
```

`internal/phpcbf` is a script that runs php_codesniffer to apply the fixes automatically, if internal/PHP_CodeSniffer/ has a local composer installation

Reply:
I tried running that (ran `composer install` before to get the local installation) but it didn't fix these automatically. I adjusted the manually for now. FWIW the code style check should/could be added to the CI in the future imo